googletest/src/gtest.cc

```diff
@@ -80,6 +80,11 @@
 
 #elif GTEST_OS_WINDOWS  // We are on Windows proper.
 
+# include <windows.h>  // NOLINT
```

Comment:
Is this include absolutely necessary? windows.h has been known to cause various issues. ( min in particular)

Reply:
This is just moved around in the file (see below). It must be included before `debugapi.h`.